### PR TITLE
20676: Updates api for new react_aggregate method

### PR DESCRIPTION
**Pull request by jackx111**

### CODE REVIEW

**Comment by jackx111** [comment]
Deleted recipes were moved previously into the recipes folder

File: tests/test_interpret_react.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def test_predicted_mean_interval(tb):
     """
     tb.inject(
         """
-        global_residual = trainee.get_prediction_stats()['moid']['mae']
+        global_residual = trainee.react_aggregate(details={"prediction_stats"=True})['moid']['mae']
```
Comment (cademack) [inline tests/test_interpret_react.py:11]: dict syntax is incorrect here, needs to be `"prediction_stats": True`

File: recipes/1-Insights/conditioned-insights.ipynb [hunk 2]
```diff
@@ -71,13 +71,13 @@
     "\n",
     "features = infer_feature_attributes(df)\n",
     "\n",
-    "# Split out the last row for a prediction set and drop the Action Feature\n",
+    "# # Split out the last row for a prediction set and drop the Action Feature\n",
```
Comment (cademack) [inline recipes/1-Insights/conditioned-insights.ipynb:74]: double comment #'s here, probably on accident?

File: recipes/1-Insights/conditioned-insights.ipynb [hunk 3]
```diff
@@ -71,13 +71,13 @@
...
     "test_case = df.iloc[[-1]].copy()\n",
     "\n",
-    "# Set the sex to be female\n",
+    "# # Set the sex to be female\n",
     "test_case.at[0, 'sex'] = 'female'\n",
     "\n",
-    "df.drop(df.index[-1], inplace=True)\n",
+    "# df.drop(df.index[-1], inplace=True)\n",
```
Comment (cademack) [inline recipes/1-Insights/conditioned-insights.ipynb:80]: should this line be removed?

File: recipes/1-Insights/engine-insights.ipynb
Comment (cademack) [inline recipes/1-Insights/engine-insights.ipynb:1]: I can't comment on the correct spot.. but the `pred_stats = ...` cell of this notebook, should maybe be one-lined?

**Review by cademack — CHANGES_REQUESTED** [review]

**Pull request merged**